### ISSUE TITLE
Discontinuity in the ET potential

### ISSUE THREAD

**Issue by Honza-R** [torchmd/torchmd-net#254]
We found that there is a discontinuity in the torchmd-net ET potential when atoms cross the cutoff distance. I assume it is not the intended behavior of the potential. We noticed that in larger calculations with our own models, but it can easily be demonstrated on a system with two atoms using a model from the SPICE-models repository (https://github.com/openmm/spice-models). Therefore, the issue is not specific to the models we are developing.

The following code (using torchmd-net 0.8.1 installed with conda, the results are the same when I upgraded to 0.11.0) reproduces the issue, there is a discontinuity of about 3.6 kJ/mol at the distance of 10A, which is the cutoff used in this model. In our models with cutoff 5 A, similar step occurs at that distance. In the intermolecular potential of larger molecules, multiple steps can be found where the individual atoms cross the cutoff.

The code produces a warning at the cutoff distance saying that gradients are not calculated, but that is not the problem - it is the change in energy.

```
import sys
import torch
from torchmdnet.models.model import load_model

# Load the model
# As an example, model from the spice-models repository is used:
# https://github.com/openmm/spice-models/blob/main/five-et/model1.ckpt
model = load_model("model1.ckpt", derivative=True)

# Types - two atoms of selected elements, H and O
types = torch.tensor([10, 21], dtype=torch.long)

# Iterate distances from 9.9 to 10.1 A
dist = 9.9
step = 0.01
while dist < 10.1:
    # Construct the geometry
    pos = torch.tensor([[0,0,0],[dist,0,0]], dtype=torch.float32)
    # Compute energy and forces
    energy, forces = model.forward(types, pos)
    # Print distance and energy
    print("dist = %.3f   energy = %.5f" % (dist, energy))
    dist += step
```

**Comment by RaulPPelaez** [comment]
Thanks for the wonderful reproducer!
We can reproduce it and are currently investigating.
